rosdep/python.yaml

```diff
@@ -1879,6 +1879,15 @@ python-grpcio:
     '*': [python-grpcio]
     bionic:
       pip: [grpcio]
+python-gsmmodem-pip: &migrate_eol_2025_04_30_python3_gsmmodem_pip
+  debian:
+    '*':
+      pip:
+        packages: [python-gsmmodem-new]
```

Comment:
It's a little odd that this is a `-new` version of this package.  Is this a fork of the original pip package?  If so, should we name the key `python3-gsmmodem-new-pip` instead?